spec/src/lib.rs

```diff
@@ -30,6 +36,11 @@ use std::sync::Arc;
 
 pub mod consensus;
 
+// Just a random secp256k1 secret key for dep group input cell's lock
+const SPECIAL_CELL_PRIVKEY: H256 =
+    h256!("0xd0c5c1e2d5af8b6ced3c0800937f996c1fa38c29186cade0cd8b5a73c97aaca3");
+const SPECIAL_CELL_CAPACITY: Capacity = capacity_bytes!(500);
```

Comment:
The capacity can be calculated if we make the dep group creation configuration. See #1435